types/shopify-buy/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for shopify-buy 1.4.0
+// Type definitions for shopify-buy 1.11.0 (WIP)
```

Comment:
Could you turn on the `dt-header` lint rule in the `tslint.json` file? This version should just be major and minor (this is automatically extracted when publishing to npm so having the correct format will prevent any issues).

Reply:
@PranavSenthilnathan great catch! Sorry I didn't see this. I should have been more thorough.

Reply:
Just updated that version reference here :) good catch @PranavSenthilnathan https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42476/files#diff-0cdb71e33d6ce9abc56e43ec7419ae4a